README.md

```diff
@@ -36,7 +36,7 @@ Artifacts can be downloaded from the [latest release](https://github.com/cerner/
 
 This library can be added as an [NPM dependency](https://www.npmjs.com/package/cucumber-forge-report-generator) via `npm i -S cucumber-forge-report-generator`
 
-This project is built on [Travis](https://travis-ci.com/cerner/cucumber-forge-report-generator/).
+This project is built on [GitHub Actions](https://github.com/cerner/cucumber-forge-report-generator/actions).
```

Comment:
We should also update the badge at the top to reference the GH workflow result: https://docs.github.com/en/actions/guides/about-continuous-integration#status-badges-for-workflow-runs